ExistingProspectRanking.cs

```diff
@@ -28,4 +30,20 @@ public ExistingProspectRanking(string rank, string chg, string name, string scho
             this.rankingDateString = date;
         }
     }
+
+    public sealed class ExistingProspectRankingCsvMap : ClassMap<ExistingProspectRanking>
+    {
+        public ExistingProspectRankingCsvMap()
+        {
+            Map(m => m.rank).Name("Rank");
+            Map(m => m.change).Name("Change");
+            Map(m => m.playerName).Name("Player");
+            Map(m => m.school).Name("School");
+            Map(m => m.position1).Name("Position");
+            Map(m => m.height).Name("Height").ConvertUsing(map => Program.convertHeightToInches(map.GetField("Height")).ToString());
```

Comment:
It looks like converting height to inches moves from the main program into here. I was initially worried that this was accidentally deleted.